### DISCUSSION TITLE
iconsole to FTMS protocol?

### DISCUSSION THREAD

**Discussion by martin-zh** [haraldh/iconsole-android#9] [Ideas]
Follow-up from https://github.com/haraldh/iconsole-android/issues/6

I found a project that converts a Kettler Bike to FTMS protocol via USB and runs on a Raspi Pi zero:
https://github.com/360manu/kettlerUSB2BLE 

Unfortunately I'm not a developer anymore so I can't make the necessary extensions so that an iconsole BT device can be used as source. Is a Raspi with 2 Bluetooth interfaces at all a possibility?

**Comment by haraldh** [comment]
>  Is a Raspi with 2 Bluetooth interfaces at all a possibility?

I read:

> The Raspberry Pi 4 Model B has an integrated Wi-Fi and Bluetooth adapter.

and additional USB bluetooth devices are possible.

**Reply by msj33** [reply]
@martin-zh 

I have started this project up:
https://github.com/msj33/iconsole_ftms
